index.html

```diff
@@ -0,0 +1,94 @@
+
+            <div class="toolbar">
+
+                    <h1 class="toolbar-logo font-family-roboto">Belden Mugauri</h1>
+
+                    <button class="menu-button" id="menu-btn" aria-controls="nav-bar">
+                        <i class="fa-solid fa-bars"><span class="sr-only">Menu</span></i>
+                    </button>
+
+  
+                    <nav id="nav-bar" data-visible="false" class="primary-navigation font-family-roboto"> 
+                        
+
+                        <ul>
+                            <li class="nav-item font-family-roboto">
+                                <a href="#">Portfolio</a>
+                            </li>
+                
+                            <li class="nav-item font-family-roboto">
+                                <a href="#">About</a>
+                            </li>
+                
+                            <li class="nav-item font-family-roboto"> 
+                                <a href="#">Contact</a> 
+                            </li>
+                   </ul>
+
+                    </nav>
+            </div> 
+
+
+
+            <div class="primary-content">
+                <img class="elipse-3" src="images\Ellipse 3.jpg" alt="">
+                <img class="elipse-4" src="images\Ellipse 4.jpg" alt="">
+                <img class="elipse-11" src="images\Ellipse 11.png" alt="">
+                <img class="elipse-12"src="images\Ellipse 12 (1).jpg" alt="">
+                <img class="elipse-13" src="images\Ellipse 12.jpg" alt="">
+                <div>
+                   <div class="primary-information">
+
+                            <h1 class="primary-title font-family-crete">I'm Belden.<br> Glad to see you!</h1>
+                            
+                            <p class="description font-family-poppins">
+                                I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                            </p>
+
+                            <span class="social-header font-family-poppins">LET'S CONNECT</span>
+                    </div>
+
+                    <div class="social-media">
+
+                            <a href="#" target="_blank" rel="noopener noreferrer">
+                                <i class="fa-brands fa-square-github"><span class="sr-only">github</span></i>
+                            </a>
+                            
+                            <a href="#" target="_blank" rel="noopener noreferrer">
+                                <i class="fa-brands fa-twitter"><span class="sr-only">twitter</span></i>
+                            </a>
+
+                            <a href="#" target="_blank" rel="noopener noreferrer">
+                                <i class="fa-brands fa-linkedin-in"><span class="sr-only">linkedin</span></i>
```

Comment:
- [ ] Kindly use `<ul>` and `<li>`  to group and list your social links. This important for semantic reasons